### PR TITLE
Update vendored dependencies for terraform 0.7.13

### PR DESCRIPTION
**Pull request by DustinChaloupka**
Updates for terraform 0.7.13.

### CODE REVIEW

**Comment by DustinChaloupka** [comment]
retest this please

**Comment by sherzberg** [comment]
:+1: :)

**Comment by adamdecaf** [comment]
👍

**Comment by DustinChaloupka** [comment]
@adamdecaf I modified the Makefile as well. Was fixing the jenkins build but it was still using old big and it's own command to test instead of what was in the repo.

**Comment by DustinChaloupka** [comment]
retest this please

**Comment by DustinChaloupka** [comment]
retest this please

**Comment by adamdecaf** [comment]
Cool. Looks good

**Comment by DustinChaloupka** [comment]
retest this please

**Comment by DustinChaloupka** [comment]
retest this please

**Comment by DustinChaloupka** [comment]
@adamdecaf I did one more change to the Makefile, and that was using a local docker-compose yaml file instead of big to run tests. It was getting hard to debug around `big` related things when it does a bunch of things this project doesn't need (like automatically adding the LB stuff). This is sort of in a direction to remove dependencies on anything Banno specific as well. Another step would be to run these tests from a container linked to the kafka scheduler so DNS isn't something to worry about either (also makes the nasty `if` statement go away), but leaving that for another PR.

File: makefile [hunk 1]
```diff
@@ -12,13 +12,21 @@ osx:
...
 
+os=$(shell uname)
+ifeq ($(os),Darwin)
+  DEV_BANNO_IP=192.168.99.100
+  LOCAL_BANNO_IP=192.168.99.1
+else
+  DEV_BANNO_IP=127.0.0.1
+  LOCAL_BANNO_IP=127.0.0.1
+endif
+
```
Comment (adamdecaf) [inline makefile]: I think this was fixed in an update to big recently. Did we want to get off big though?

Comment (DustinChaloupka) [inline makefile]: This isn't using big, just what big had configured for the DNS container (which we can remove, mentioned in the other comment).

**Pull request merged**